chapter_5_practic/task11.cpp

```diff
@@ -0,0 +1,71 @@
+﻿// Write a program, based on the sterling structure of Exercise 10 in Chapter 4, that
+// obtains from the user two money amounts in old - style British format(£9:19 : 11), adds
+// them, and displays the result, again in old - style format.Use three functions.The first
+// should obtain a pounds - shillings - pence value from the user and return the value as a
+// structure of type sterling.The second should take two arguments of type sterling and
+// return a value of the same type, which is the sum of the arguments.The third should take
+// a sterling structure as its argumentand display its value
+
+#include <iostream>
+
+struct sterling
+{
+	int pounds = 0;
+	int shillings = 0;
+	int pence = 0;
+};
+
+sterling user_input();
+sterling total(sterling a, sterling b);
+void output(sterling result);
+
+int main()
+{
+	output(total (user_input(), user_input()));
+	
+	return EXIT_SUCCESS;
+}
+
+sterling user_input()
+{
+	int user_pounds = 0;
+	int user_shillings = 0;
+	int user_pence = 0;
+	char ch = ' ';
+
+	std::cout << "Enter the amount of money (e.g. 5.10.6): ";
+	std::cin >> user_pounds >> ch >> user_shillings >> ch >> user_pence;
+	sterling from_user = { user_pounds, user_shillings, user_pence };
+
+	return from_user;
+}
+
+sterling total(sterling a, sterling b)
+{
+	int total_pounds = 0;
+	int total_shillings = 0;
+	int total_pence = 0;
+	
+	total_pence = a.pence + b.pence;
+	while (total_pence > 11)
+	{
+		total_pence -= 12;
+		total_shillings += 1;
+	}
+
+	total_shillings += (a.shillings + b.shillings);
+	while (total_shillings > 19)
+	{
+		total_shillings -= 20;
+		total_pounds += 1;
```

Comment:
`total_pounds = total_shillings / 20`

Reply:
Fixed in https://github.com/Dmitriy-Romanchuk/master/commit/201bff4b43170612ef65fe55bd4823f0d276f381